packages/swc-plugin-workflow/transform/src/naming.rs

```diff
@@ -6,3 +6,32 @@ use std::fmt::Display;
 pub fn format_name(prefix: &str, filepath: &str, function_name: impl Display) -> String {
     format!("{prefix}//{filepath}//{function_name}")
 }
+
+#[cfg(test)]
```

Comment:
@ijjk wdyt?

Reply:
doesn't look like it's actually testing backslash below but the place it's normalizing looks right